spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -38,14 +38,32 @@
       expect(user.sms?).to eq true
     end
 
-    context 'with number that does not support phone delivery method' do
+    context 'user enters OTP incorrectly 3 times' do
```

Comment:
I'm like I'm seeing a ghost. I deleted this test in a PR that was merged while this was open. We should be able to remove it

Reply:
done